# An array of PointStamped points
multi_explore/Frontier[] frontiers